doc-content/enterprise-only/employee-rostering/er-spots-proc.adoc

```diff
@@ -7,17 +7,12 @@ For each spot, you can select one or more required skills from the list that you
 
 .Procedure
 
-. To enter or change spots in the Optaweb employee rostering application web interface, click the *Spots* tab.
-+
-You can see the numbers of currently visible spots in the top right part of the browser window, for example, `1-15 of 34`. You can use the *<* and *>* buttons to display other spots in the list.
-+
-You can enter any part of a spot name in the *Search* box to search for spots.
-+
+. To enter or change spots in the OptaWeb employee rostering application web interface, click the *Spots* tab. You can enter any part of a spot name in the *Search* box to search for spots.
 . Complete the following steps to add a new spot:
 .. Click *Add Spot*.
-.. Enter the name of the new spot in the text field under *Spot name*.
+.. Enter the name of the new spot in the text field under *Name*.
 .. Optionally, select one or more skills from the drop-down list under *Required skill set*.
-.. Click *Save Spot*.
+.. Click the Save icon.
```

Comment:
Question - does this use a floppy disk icon or is it titled?

Suggestion - bold name if titled or use lowercase if not:

s/Click the Save icon./Click the save icon/Click the *Save* icon.

Reply:
Icon with no title, same as Employee and Skill

Reply:
Yeah.. my rational was even though it's not labelled, it's still name, just like you are Clifton whether I write it or put it in the UI. So if it was labelled or hover text, I would bold because a UI element.